lib/cocoapods-search/command/search.rb

```diff
@@ -82,7 +84,19 @@ def local_search
           sets.reject! { |set| !set.specification.available_platforms.map(&:name).include?(platform) }
         end
 
-        sets.reverse_each do |set|
+        if @use_pager
+          IO.popen((ENV['PAGER'] || 'less -R'), 'w') do |io|
```

Comment:
extract this into a `with_pager` method that handles the `popen`, trapping, and setting the `output_io` and yields?